### ISSUE TITLE
[feature] speed up pickup by allowing to press number key instead of drag

### ISSUE THREAD

**Issue by msouth** [ironbane/IronbaneServerLegacy#381]
When adding a pickup item to inventory, it would be nice if you could hit a number key to say which inventory slot you would like the item to go to.  Instead of having to drag.

**Comment by bsparks** [comment]
how about double click on loot item and having it go into first available slot

**Comment by msouth** [comment]
absolutely!  I had thought that too, and would probably be easier to implement, I would guess.

Although the number thing might still be nice if combined with #382 when you have a full inventory.

Or a full inventory could just prompt. You double click, it says "what do you want me to drop?" if your inventory is full.

**Comment by msouth** [comment]
After playing again, I still think the number thing is a good idea.  Faster than double clicking, and doesn't require you to re-aim your mouse pointer if you are in the heat of battle.  Which is when speed matters most.

Of course, some bags have more than one thing....so that would be ambiguous in that case, you would be specifying a destination but not a source....

Haptic sensor support, it seems, is what I'm ultimately looking for :).

A dialog box that came up during a battle could kill you.
